README.md

```diff
@@ -59,6 +59,7 @@ Stay up to date with the latest ONNX news. [[Facebook](https://www.facebook.com/
 numpy >= 1.16.6
 protobuf >= 3.12.2
```

Comment:
I can provide the history context [here](https://github.com/onnx/onnx/pull/3684): 3.12.2 is the minimum supported Protobuf version for the latest ONNX (as mentioned in [requirements.txt](https://github.com/onnx/onnx/blob/main/requirements.txt)) and 3.16.0 is the suggested version (also what the release CIs are using now). We didn't set 3.16.0 as the minimum supported version because many projects rely on ONNX and we want to provide some flexibility to support lower Protobuf version. At least for current ONNX, 3.12.2+ Protobuf can work, but Protobuf<=3.11 cannot.

Reply:
OK, I still think having this list here (beyond the Python version) is not helpful. It's redundant with requirements.txt.